html5/verto/js/src/jquery.jsonrpcclient.js

```diff
@@ -322,8 +323,9 @@
                         self.options.onWSClose(self);
                     }
 
-                    if (self.ws_cnt > 10 && self.options.wsFallbackURL) {
-                      self.options.socketUrl = self.options.wsFallbackURL;
+                    if (self.ws_cnt > ((self.ws_fallback_cnt + 1) * 10) && self.options.wsFallbackURL && self.options.wsFallbackURL.length > 0) {
```

Comment:
`self.options.wsFallbackURL` could be a string. I'd suggest to force it to be an array or - at least - check the type.